### ISSUE TITLE
[Bug]: PyTorch Quickstart - avg_trainloss scales with local-epochs > 1

### ISSUE THREAD

**Issue by jun-simons** [adap/flower#6333]
### Describe the bug

In quickstart-pytorch, the reported `avg_trainloss` increases linearly with `local-epochs`.  Upon searching this issue was pointed out [in the discussion forum](https://discuss.flower.ai/t/i-was-reviewing-the-train-function-in-the-quickstart-pytorch-example-and-noticed-a-logical-issue-in-how-the-average-training-loss-is-calculated/1159).

The fix is simply to include the number of epochs when normalizing.

### Steps/Code to Reproduce

From Quickstart, run with different numbers of epochs:
`flwr run . --run-config "num-server-rounds=1 local-epochs=1"` : train_loss 2.21
`flwr run . --run-config "num-server-rounds=1 local-epochs=2"` : train_loss = 4.34
`flwr run . --run-config "num-server-rounds=1 local-epochs=4"` : train_loss = 8.57

### Expected Results

Average loss should not increase linearly with the number of epochs as it should be appropriately normalized

### Actual Results

1 epoch: train_loss = 2.21
2 epochs: train_loss = 4.34 (~x2)
4 epochs: train_loss = 8.57 (~x4)
